public/app/themes/justice/error-pages/maintenance.html

```diff
@@ -4,52 +4,16 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
-
-  <title>Website temporarily unavailable | Intranet</title>
-  <link rel="stylesheet" href="/app/themes/clarity/dist/css/globals.css"/>
-  <link rel="stylesheet" href="/app/themes/clarity/dist/css/style.css"/>
-
-  <!-- Google Tag Manager -->
-  <noscript>
-    <iframe src="//www.googletagmanager.com/ns.html?id=GTM-P545JM" height="0" width="0"
-            style="display:none;visibility:hidden"></iframe>
-  </noscript>
-  <script>
-    (function (w, d, s, l, i) {
-      w[l] = w[l] || []
-      w[l].push({
-        'gtm.start': new Date().getTime(),
-        event: 'gtm.js'
-      })
-      var f = d.getElementsByTagName(s)[0],
-        j = d.createElement(s),
-        dl = l != 'dataLayer' ? '&l=' + l : ''
-      j.async = true
-      j.src =
-        '//www.googletagmanager.com/gtm.js?id=' + i + dl
-      f.parentNode.insertBefore(j, f)
-    })(window, document, 'script', 'dataLayer', 'GTM-P545JM')
-
-  </script>
-  <!-- End Google Tag Manager -->
-
+  <title>Website temporarily unavailable | Justice UK</title>
+  <link rel="stylesheet" href="/app/themes/justice/src/css/global.css"/>
 </head>
 
 <body>
 <header class="c-logo-bar">
   <div class="u-wrapper">
-
     <div class="u-wrapper__stack--left">
-      <a href="/" rel="home">
-        <img src="/app/themes/clarity/dist/images/moj_logo.png" alt="MoJ Logo">
-        <span class="agency-title l-half-section">Ministry of Justice intranet</span>
-      </a>
-    </div>
-
-    <div class="u-wrapper__stack--right">
-      <a href="/agency-switcher" class="c-logo-bar__switch">Switch to other intranet</a>
+      <img src="/app/themes/justice/src/img/moj-logo.gif" alt="MoJ Logo">
```

Comment:
Can we reference `dist`?

**The process is...**
LaravelMix should process the image from `src`, apply compressions if necessary and place the image in the `dist` directory structure for use in production. `src` isn't there 👍 

![Screenshot 2024-05-30 at 13 11 28](https://github.com/ministryofjustice/justice-gov-uk/assets/1674630/d8814206-2889-4c7d-8b27-a1389ac5272b)